src/test/java/core/basesyntax/services/impl/CsvReaderImplTest.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax.services.impl;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class CsvReaderImplTest {
+    private static final String VALID_PATH = "src/test/resources/testRead.csv";
+    private static final String INVALID_PATH = "";
+    private static final String FILE_CONTENT = "This is a file for test.";
+    private CsvReaderImpl csvReader;
+
+    @BeforeEach
+    void setUp() {
+        csvReader = new CsvReaderImpl();
+    }
+
+    @Test
+    void testRead_Ok() {
```

Comment:
Check again tests naming rules and pay attention to lowerCamelCase rule. In this case, I would recomment to name this test like "read_validPath_ok"